modules/administration/pages/openscap.adoc

```diff
@@ -74,10 +74,14 @@ You are advised to use the [package]``scap-security-guide-redhat`` package you w
 In the current version of [package]``scap-security-guide``, {suse} supports the following profiles:
 
 * DISA STIG profile for {sles} 12 and 15
+* ANSSI-BP-028 profile for {sles} 12 and 15
 * PCI-DSS profile for {sles} 12 and 15
 * HIPAA profile for {sles} 12 and 15
+* CIS profile for {sles} 12 and 15
+* Hardening for Public Cloud Image of {sles} for SAP Applications 15
+* Public Cloud Hardening for {sle} 15
 
```

Comment:
Standard System Security profile for SLES 15 and 12 is missing from the list.

References (see top right drop-down in the following pages):
- https://static.open-scap.org/ssg-guides/ssg-sle15-guide-index.html
- https://static.open-scap.org/ssg-guides/ssg-sle12-guide-index.html